xenit-alfresco/templates/acs/acs-hpa.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.acs.hpa.enabled }}
+{{- if and (.Values.acs.hpa.enabled) (.Values.general.enterprise) }}
```

Comment:
this hpa should be removed completely as it not fully functional from the start

Reply:
done